python_modules/dagster/dagster/core/storage/sql.py

```diff
@@ -92,6 +98,8 @@ def run_migrations_offline(context, config, target_metadata):
     script output.
 
     """
+    from sqlite3 import DatabaseError
```

Comment:
We do some exception handling shenanigans here for the SQLite connection... seemed like it was harmless to put in our generic alembic migration path.